docs/testing/uninstall-add-in.md

```diff
@@ -0,0 +1,142 @@
+- **Teams Toolkit CLI**: Use the `teamsapp uninstall` command at the same command line where you started the session with `teamsapp install`. For more information, see [Sideload with the Teams Toolkit CLI](../develop/convert-xml-to-json-manifest.md#sideload-with-the-teams-toolkit-cli-command-line-interface).
+- **Visual Studio**: Select **DEBUG** | **Stop debugging** in the menu, or press SHIFT-F5, or click the square red "stop" button on the debugging bar. Alternatively, closing the Office application also stops the session and uninstalls the add-in. For more information, see [First look at the Visual Studio debugger](/visualstudio/debugger/debugger-feature-tour).
+
+## Remove a ghost add-in
+
+To remove a ghost add-in, you need to remove the artifacts that were created when it was last sideloaded, remove it's local registration, and for Outlook add-ins remove it's registration in Exchange.
+
+The process for removing the add-in varies depending on whether the add-in is for Outlook or some other Office application. 
+
+> [!NOTE]
+> In the [unified manifest for Microsoft 365](../develop/unified-manifest-overview.md), an add-in can be configured to support Outlook and one or more other Office applications; that is, there is more than one member of the "extensions.requirements.scopes" array in the manifest and one of the members is "mail" (or the "extensions.requirements.scopes" property isn't present). Treat an add-in that is configured in this way as an Outlook add-in.
+
+If the ghost add-in is not an Outlook add-in, skip to the section [Remove the add-in artifacts](#remove-the-add-in-artifacts).
+
+### Remove the Exchange registration of a ghost Outlook add-in
+
+1. Be sure you are logged into Outlook with the same ID you used when you sideloaded the add-in.
+1. Open PowerShell as an Administrator.
+1. Run the following commands. Answer "Yes" to all confirmation prompts.
+
+   ```powershell
+   Install-Module -Name ExchangeOnlineManagement -RequiredVersion 3.4.0
+   Set-ExecutionPolicy RemoteSigned
+   Connect-ExchangeOnline
+   ```
+
+      > [!NOTE]
+      > If the `Connect-ExchangeOnline` command returns the error "ActiveX control '8856f961-340a-11d0-a96b-00c04fd705a2' cannot be instantiated because the current thread is not in a single-threaded apartment", just run the command a second time. This is a well-known bug.
+
+1. Run the following command. Answer "Yes" to all confirmation prompts.
+
+   ```powershell
+   Get-App | Format-Table -Auto DisplayName,AppId
+   ```
+
+   A list of the add-ins installed on Outlook displays. Most of them are built-in Microsoft add-ins, but if you have installed add-ins from other companies, they are listed too as are any ghost Outlook add-ins. 
```

Comment:
Missing comma, but consider rewriting that second sentence to avoid so many commas (maybe split it into two?)
```suggestion
   A list of the add-ins installed on Outlook displays. Most of them are built-in Microsoft add-ins, but if you have installed add-ins from other companies, they are listed too, as are any ghost Outlook add-ins. 
```